### PR TITLE
RANCHER-1846. Fix some modules' okapi integration part.

### PR DESCRIPTION
**Pull request by epam-avramenko**

### CODE REVIEW

File: resources/helm/development.yaml [hunk 1]
```diff
@@ -1670,7 +1673,10 @@ mod-ldp:
     maxUnavailable: 1
 mod-reporting:
   extraEnvVars: []
-  integrations: {}
+  integrations:
+    okapi:
```
Comment (OHaimanov) [inline resources/helm/development.yaml:1677]: mod-reporting, don't need this integration

File: resources/helm/development.yaml [hunk 2]
```diff
@@ -2338,6 +2356,9 @@ mod-quick-marc:
 mod-reading-room:
   extraEnvVars: []
   integrations:
+    okapi:
```
Comment (OHaimanov) [inline resources/helm/development.yaml:2359]: mod-reading-room don't need this integration

File: resources/helm/development.yaml [hunk 3]
```diff
@@ -2364,6 +2385,9 @@ mod-reading-room:
 mod-record-specifications:
   extraEnvVars: []
   integrations:
+    okapi:
```
Comment (OHaimanov) [inline resources/helm/development.yaml:2388]: mod-record-specifications don't need okapi integration

**Review by OHaimanov — CHANGES_REQUESTED** [review]

File: resources/helm/development.yaml (hunk 1 above)
Comment (epam-avramenko) [reply resources/helm/development.yaml]: Done

File: resources/helm/development.yaml (hunk 3 above)
Comment (epam-avramenko) [reply resources/helm/development.yaml]: Done

File: resources/helm/development.yaml (hunk 2 above)
Comment (epam-avramenko) [reply resources/helm/development.yaml]: Done

**Review by aatoyan — APPROVED** [review]

**Review by OHaimanov — APPROVED** [review]

**Pull request merged**